prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -139,13 +141,25 @@ class FakeDataService {
                         )
                 }
                 element.table?.startsWith("LIVD-Supplemental") == true -> {
-                    if (element.tableColumn == null) return ""
+                    if (element.tableColumn == null)
+                        return ""
                     element.default ?: ""
                 }
                 element.table == "fips-county" -> {
                     when {
                         element.nameContains("state") -> context.state
                         element.nameContains("county") -> context.county
+                        (element.default == null) -> ""
+                        else -> TODO("Add this column in a table")
```

Comment:
This should not be a `TODO` but should be a warning and then return null. @carlosfelix2 do you agree? I think if we make it an error then we could halt processing for a bad column name. Thoughts?

Reply:
Done